pkg/apis/planetscale/v2/vitesscell_types.go

```diff
@@ -67,6 +67,13 @@ type VitessCellSpec struct {
 	// in which case this field is automatically managed by the VitessCluster
 	// controller that owns this VitessCell.
 	Images VitessCellImages `json:"images,omitempty"`
+
+	// ExtraVitessFlags can optionally be used to override default flags set by the
```

Comment:
Since this is outside the `VitessCellTemplate`, users can't set it, which is good because this is meant for internal use. In those cases, I prefer to keep the doc comment aimed at an internal audience:

```go
// ExtraVitessFlags is inherited from the parent's VitessClusterSpec.
```

Reply:
Noted - thanks for the tip.  Will adjust now.

Reply:
Done https://github.com/planetscale/vitess-operator/pull/14/commits/226927d4cabfc3a79db1fcea1f2079d61b4ec302.